boards/arm/rddrone_fmuk66/rddrone_fmuk66.dts

```diff
@@ -363,6 +363,16 @@ zephyr_udc0: &usbotg {
 		status = "okay";
 	};
 
+	fxas21002@1 {
```

Comment:
CS1 needs to be added in this commit

Reply:
Ok, Acknowledged

Reply:
@gmarull .. We have made your suggested changes.. kindly check again